backend/apps/notifications/tasks.py

```
@@ -108,6 +133,18 @@ def send_bulk_email(payload):
         )
+        try:
+            html_message = render_to_string(
+                "notifications/badge_email.html",
+                {
+                    "title": f"Badge Unlocked: {badge_name}",
+                    "message": f"Congratulations {username}! You earned the '{badge_name}' badge.",
+                    **data,
+                },
+            )
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)